### ISSUE TITLE
mutagen does not check for valid tag size <= 28 bits

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dhamaniasad/mutagen#150]
```
ID3 spec states that header size should only use 28 of the 32 bits in the size 
field i.e. size <=256MB.

mutagen id3.py __load_header does not check for valid size (at least if 
PEDANTIC is True).
```

Original issue reported on code.google.com by `Sophist...@gmail.com` on 25 May 2013 at 8:54

**Comment by GoogleCodeExporter** [comment]
```
Not 100% correct: mutagen will ignore those 4 bits (but doesn't check if they 
are zero aka synchsafe)

Any particular reason why you want this to be more strict?
```

Original comment by `reiter.christoph@gmail.com` on 4 Sep 2013 at 11:39
- Changed state: **NeedInfo**

**Comment by GoogleCodeExporter** [comment]
```
Well if PEDANTIC is True, I think it should probably check.
```

Original comment by `Sophist...@gmail.com` on 4 Sep 2013 at 12:17

**Comment by GoogleCodeExporter** [comment]
```
Fair enough.
```

Original comment by `reiter.christoph@gmail.com` on 4 Sep 2013 at 12:47
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 46a4ac82d8fa.
```

Original comment by `reiter.christoph@gmail.com` on 4 Sep 2013 at 3:15
- Changed state: **Fixed**
